tests/README.md

```diff
@@ -76,6 +76,16 @@ tmt run
 This will use latest BlueChi packages from
 [hirte-snapshot](https://copr.fedorainfracloud.org/coprs/mperina/hirte-snapshot/) repository.
 
+## Running integration tests with memory leak detection
+
+To run integration tests with `valgrind`, set `RUN_WITH_VALGRIND` environment variable as follows:
+
+```shell
+RUN_WITH_VALGRIND=1 tmt run -v
```

Comment:
Just wondering if `WITH_VALGRIND` wouldn't be descriptive enough, but shorter

Reply:
Yep, and more elegant